src/SearchForm/SearchForm.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react';
+import './SearchForm.css'
+
+export default class SearchForm extends Component {
+	constructor(props){
+		super(props);
+		this.state = {searchString : ''}
+	}
+	
+	handleChange = (e) => {
+ 	  this.setState({searchString : e.target.value});
+	}
+
+	//создаем елементы выпадающего списка фильмов
+	createList = (item, index) =>{
+		return(
+			<li key={item.name}>
+				<a href={item.url}>{item.name}</a>
+			</li>
+		)
+	}
+
+	render(){
+		let libraries = this.props.items,
```

Comment:
Please move logic from render fn to other fn or into appropriate lifecycle hook.